src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -624,30 +624,32 @@ def _create_venv_script(
             #!{bash.path}
             set -euo pipefail
 
-            # N.B.: We convert all sandbox root relative paths to absolute paths so this script
-            # works when run with a cwd set elsewhere.
-
             # N.B.: This relies on BASH_SOURCE which has been available since bash-3.0, released in
-            # 2004. In turn, our use of BASH_SOURCE relies on the fact that this script is executed
-            # by the engine via its absolute path.
-            ABS_SANDBOX_ROOT="${{BASH_SOURCE%/*}}"
+            # 2004. It will either contain the (absolute or relative) path of the venv script, which
```

Comment:
Thanks for clarifying the docs. This is still not completely clear: For one thing, it looks like the parentheses aren't called for, since omitting the parenthesized clause makes the sentence not make sense. And also, it would be good to know what the "relative path of the venv script" is relative to (presumably the CWD)?